frontend/www/js/omegaup/components/problem/CollectionList.vue

```diff
@@ -78,10 +101,26 @@ export default class CollectionList extends Vue {
   @Prop() sortOrder!: string;
   @Prop() columnName!: string;
   @Prop() difficulty!: string;
+  @Prop({ default: () => [] }) selectedTags!: string[];
 
   T = T;
   level = this.data.level;
-  tags: string[] = this.data.frequentTags.map((element) => element.alias);
+
+  get availableTags(): string[] {
+    if (this.currentTags.length == 0) {
+      return this.data.frequentTags.map((element) => element.alias);
+    } else {
+      let tags: string[] = this.data.frequentTags.map(
+        (element) => element.alias,
+      );
+      this.currentTags.forEach((element) => {
+        if (!tags.includes(element)) {
+          tags.push(element);
+        }
+      });
+      return tags;
```

Comment:
se puede usar un [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) para hacer más rápido L117?

```suggestion
      let tags: Set[string] = new Set(
        this.data.frequentTags.map((element) => element.alias)
      );
      this.currentTags.forEach((element) => tags.add(element));
      return Array.from(tags);
```